src/Core/Components/List/FluentOption.razor.cs

```diff
@@ -25,14 +24,14 @@ public partial class FluentOption<TOption> : FluentComponentBase, IDisposable wh
     /// Gets or sets a value indicating whether the element is selected.
     /// </summary>
     [Parameter]
-    public bool Selected { get; set; }
+    public bool? Selected { get; set; }
```

Comment:
Why `Disabled` and `Selected` are nullable? In my mind, a value can be selected or not (same for a disabled value).

Reply:
If they are null, they will not be rendered by the web component as attributes so web component can apply its default rendering

Reply:
I don't think so. 

Example: 
```razor
<h3 my-flag="@MyFlag">MyComponent</h3>

@code {
    [Parameter]
    public bool MyFlag { get; set; }
}
```

Will generate these code: `my-flag` will be generated only when the value if True.

```html
<MyComponent />                    ->   <h3>MyComponent</h3>
<MyComponent MyFlag="true" />      ->   <h3 my-flag>MyComponent</h3>
<MyComponent MyFlag="false" />     ->   <h3>MyComponent</h3>
```

Reply:
Right, bool is different that way (comared to string value). Will revert.